scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
   MIN_SIZE: 569,
-  MAX_SIZE: 843,
+  MAX_SIZE: 850,
```

Comment:
The bundle size has jumped by quite a lot (`847`on my machine), idk if it's related to the new psammead-styles package update @simonsinclair ?

Reply:
No, `psammead-styles` changes were only replacements and it shouldn't have been pulled into Simorgh yet.

Reply:
From the bundle report for this PR - the social-embed bundle includes psammead styles
<img width="802" alt="image" src="https://user-images.githubusercontent.com/9645462/100241761-f8d8f780-2f2b-11eb-8419-e830efc4fe05.png">

From the bundle report for latest - the social embed bundle does not include psammead styles:
<img width="802" alt="image" src="https://user-images.githubusercontent.com/9645462/100241920-232ab500-2f2c-11eb-8109-c72f4aba85e9.png">

I think this is okay temporarily as it will be optimised when nassim is removed from latest and the new psammead styles package is used more instead across Simorgh, does this make sense @simonsinclair ?